### ISSUE TITLE
Handle saving that takes too long

### ISSUE THREAD

**Issue by aldnav** [uncaughtxcptn/endpoints#81]
Tried a case to put in a media response.
Can prompt an error message or have a max timeout.

![image](https://user-images.githubusercontent.com/2098694/36769803-a21bc3a6-1c81-11e8-8706-cde68f122465.png)

**Comment by aldnav** [comment]
@arnellebalane that's what I was trying to do. That case above is for a photo.

**Comment by name3anad** [comment]
@arnellebalane @aldnav I don't this we should support storing media content. It can be very costly

**Comment by aldnav** [comment]
@name3anad yes I agree. Let's just set a timeout or something just to keep the page responsive. 👍
